src/main/java/de/tum/in/www1/artemis/repository/plagiarism/PlagiarismCaseRepository.java

```diff
@@ -134,4 +134,6 @@ default PlagiarismCase findByIdWithPlagiarismSubmissionsElseThrow(long plagiaris
     default PlagiarismCase findByIdElseThrow(long plagiarismCaseId) {
         return findById(plagiarismCaseId).orElseThrow(() -> new EntityNotFoundException("PlagiarismCase", plagiarismCaseId));
     }
+
+    long countByExerciseId(long exerciseId);
```

Comment:
Are there any edge cases for removed/disabled entities (eg. disabled/deleted Users) ?

Reply:
Not any I'm aware of that should break anything.
We never delete users but only anonymize them, so the worst thing could happen is that we count a potential plagiarism result for a deleted user and I don't think this will happen often, so I would postpone it to a follow-up if we want to consider this case specifically.